src/components/views/Header.js

```diff
@@ -11,18 +14,68 @@ import "styles/views/Header.scss";
  * https://reactjs.org/docs/components-and-props.html
  * @FunctionalComponent
  */
-const Header = props => (
-  <div className="header container" style={{height: props.height}}>
-    <h1 className="header title">Group 07 in SoPra FS22 rocks with React!</h1>
-    <ReactLogo width="60px" height="60px"/>
-  </div>
-);
 
+function Header(props){
+    const history = useHistory();
+
+    const goToRegistration = async () => {
+        // do Registration instead of login
+        try{
+            history.push('/registration')
+        } catch (error){
+            alert(`Failed to find Register Page: \n${handleError(error)}`);
+        }
+    };
+
+    const goToLogin = async => {
+        // do login instead of registration
+        try{
+            history.push('/login')
+        } catch (error){
+            alert(`Failed to find Login Page: \n${handleError(error)}`);
+        }
+    };
+
+    if(!localStorage.getItem("token")){
+        if(props.view=="login"){
+            return     <div className="header container" style={{height: props.height}}>
+                <h1 className="header title">Group 07 in SoPra FS22 rocks with React!</h1>
+                <ReactLogo width="60px" height="60px"/>
+                <div className="header button-container">
+                    <Button
+                            onClick={() => goToRegistration()}
+                    >
+                        Register Account
+                    </Button>
+                </div>
+            </div>
+        }
+        if(props.view=="register"){
+            return     <div className="header container" style={{height: props.height}}>
```

Comment:
Why don't we just write height: 100 here? It should not change throughout the entire website?